docs/maintain-guides-how-to-validate-kusama.md

```diff
@@ -133,12 +149,11 @@ if you do not want to start in validator mode right away.
 
 **Note:** The `--pruning=archive` flag is implied by the `--validator` and `--sentry` flags, so it is only required explicitly if you start your node without one of these two options. If you do not set your pruning to archive node, even when not running in validator and sentry mode, you will need to re-sync your database when you switch.
 
-#### Previous Kusama CC1 Validator
+#### Previous Kusama CC2 Validator
 
-Before synchronizing the the CC2 chain data, you will need to copy your previous keystore to the new chain id in order to
-use the previously set session keys. If you do not do this, you will need to generate and set your session keys again.
+Before synchronizing the latest chain data, you will need to copy your previous keystore to the new chain id in order to use the previously set session keys. If you do not do this, you will need to generate and set your session keys again.
 
-Kusama CC1 and Kusama CC2 have different default data directories, usually located in the `$HOME/.local/share/polkadot/chains` directory on Linux based machines. For example, the default directory for Kusama CC1 is `$HOME/.local/share/polkadot/chains/ksma/keystore` while CC2 keys are located in `$HOME/.local/share/polkadot/chains/ksmcc2/keystore`.
+Kusama CC1 and Kusama CC2 have different default data directories, usually located in the `$HOME/.local/share/polkadot/chains` directory on Linux based machines. For example, the default directory for Kusama CC2 is `$HOME/.local/share/polkadot/chains/ksmcc2/keystore` while CC2 keys are located in `$HOME/.local/share/polkadot/chains/ksmcc2/keystore`.
```

Comment:
Do we really still need to go into depth about the CC1 and CC2 chains?  We've been using CC3 since at least October/November.  Are nodes still going to be upgrading from them?

Reply:
We can remove that but we could imagine once Polkadot launch, validators will be also doing the same thing, so I guess it would be worth to stay here for reference.

Reply:
Let's move it to it's own page on the Kusama guide? That way it's not directly in the guide but can still be accessible.

Reply:
sounds good